sdks/csharp/model_operations.cs

```diff
@@ -2,10 +2,8 @@
 // <CreateModel>
 // 'client' is an instance of DigitalTwinsClient
 // Read model file into string (not part of SDK)
-using var modelStreamReader = new StreamReader("MyModelFile.json");
-string dtdl = modelStreamReader.ReadToEnd();
-string[] dtdls = new string[] { dtdl };
-client.CreateModels(dtdls);
+string dtdl = new File.ReadAllText("MyModelFile.json");
```

Comment:
File.ReadAllText is fewer lines than using a StreamReader, requires no disposing.